### DISCUSSION TITLE
TypeScipt initialization

### DISCUSSION THREAD

**Discussion by rodionveretin** [eta-dev/eta#270] [Q&A] [answered]
I import ETA as in the documentation, but it gives me the error "ayvavy". What Im doing wrong?

`
import { Eta } from 'eta';
import * as path from 'path';
const eta = new Eta({ views: path.join(__dirname, "templates") })
`

src/app.ts:3:21 - error TS2554: Expected 0 arguments, but got 1.
3 const eta = new Eta({ views: path.join(__dirname, "templates") })

**Comment by bgub** [comment]
@rodionveretin what version of Eta are you using?

**Reply by rodionveretin** [reply]
3.2.0

**Reply by rodionveretin** [reply, marked as answer]
It was a problem with the typescript version, I installed the latest one, the error went away
